src/route.rs

```diff
@@ -126,6 +126,55 @@ impl Route {
         self
     }
 
+    /// Set the service to be called when this route is hit.
```

Comment:
I think `service` must be a link to a structure. Otherwise, this comment doesn't explain much.

Reply:
Eh, the comment is more of a header, the code explains more. If it linked to `Service` it would be weird because the method takes a `ServiceFactory`, and if it linked to `ServiceFactory` it would be weird because it isn't really meant for service factories, it's meant for custom services. The comment is consistent with other `Route` methods docs so :man_shrugging: